MakeHist/makeHists.py

```diff
@@ -0,0 +1,61 @@
+from os import listdir, mkdir, popen
+from os.path import isfile, join, exists
+from sampleDict import sampleDict
+from ROOT import *
+
+
+gSystem.CompileMacro("mumu.C", "gOck")
+gSystem.Load("mumu_C")
+
+def makePlots(l_sampleDict, lumi, cutString):
+  chain = TChain('SimpleNtupler/t')
+  inFiles = []
+
+  for path in l_sampleDict['path']:
+    for inFile in filter(None,popen("ls -u "+path).read().split('\n')):
+      if ".root" in inFile and ".root" not in path:
+        pathString = path  + inFile
+      elif ".root" in path:
+        pathString = path
+
+
+      chain.Add(pathString)
+
+  instance = mumu(chain)
+
+  if not exists("output"):
+    mkdir("output")
+  instance.Loop("output/{0}_{1}".format(cutString,l_sampleDict['outName']),l_sampleDict['xsection'],lumi,l_sampleDict['totalEvents'], cutString)#
+
+
+cutStrings = ["b=1_j=0",
+"b=1,2_b+j=2"]
+
+cutStrings = ["1",
+"2",
+"3",
+"4",
+"5",
+""]
+
```

Comment:
Why do you have two lists here that don't append, but rather overwrite one another? Unless I misunderstand python here.